_targets/meta/meta

```diff
@@ -0,0 +1,53 @@
+name|type|data|command|depend|seed|path|time|size|bytes|format|repository|iteration|parent|children|seconds|warnings|error
```

Comment:
What is this file for, should it be committed?

Reply:
this is a log of `target` runs, I've looked into it a bit and it looks like it should be committed